src/ResourceManager/RedisCache/Commands.RedisCache/Models/RedisCacheClient.cs

```diff
@@ -41,11 +41,11 @@ public RedisCacheClient(AzureContext context)
         }
         public RedisCacheClient() { }
 
-        public RedisResourceWithAccessKey CreateOrUpdateCache(string resourceGroupName, string cacheName, string location, string skuFamily, int skuCapacity, string skuName,
+        public RedisResource CreateCache(string resourceGroupName, string cacheName, string location, string skuFamily, int skuCapacity, string skuName,
```

Comment:
> public RedisResource CreateCache(string resour [](start = 7, length = 47)

@SiddharthChatrolaMs we are no longer taking breaking changes in PS. The best we can do is, add a warning for the breaking change and then in about 6 months commit the breaking change in one of the releases.
Please also help us understand the reason for this breaking change and why it cannot be address with a workaround (in this case an overload) or why it cannot wait for 6 months for the next major release of PS.